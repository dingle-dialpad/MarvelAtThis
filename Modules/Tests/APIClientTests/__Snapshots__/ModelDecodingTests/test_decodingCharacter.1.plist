<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>code</key>
	<integer>200</integer>
	<key>data</key>
	<dict>
		<key>count</key>
		<integer>1</integer>
		<key>limit</key>
		<integer>20</integer>
		<key>offset</key>
		<integer>0</integer>
		<key>results</key>
		<array>
			<dict>
				<key>comics</key>
				<dict>
					<key>available</key>
					<integer>12</integer>
					<key>collectionURI</key>
					<dict>
						<key>relative</key>
						<string>http://gateway.marvel.com/v1/public/characters/1011334/comics</string>
					</dict>
					<key>items</key>
					<array>
						<dict>
							<key>name</key>
							<string>Avengers: The Initiative (2007) #14</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/comics/21366</string>
						</dict>
						<dict>
							<key>name</key>
							<string>Avengers: The Initiative (2007) #14 (SPOTLIGHT VARIANT)</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/comics/24571</string>
						</dict>
						<dict>
							<key>name</key>
							<string>Avengers: The Initiative (2007) #15</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/comics/21546</string>
						</dict>
						<dict>
							<key>name</key>
							<string>Avengers: The Initiative (2007) #16</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/comics/21741</string>
						</dict>
						<dict>
							<key>name</key>
							<string>Avengers: The Initiative (2007) #17</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/comics/21975</string>
						</dict>
						<dict>
							<key>name</key>
							<string>Avengers: The Initiative (2007) #18</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/comics/22299</string>
						</dict>
						<dict>
							<key>name</key>
							<string>Avengers: The Initiative (2007) #18 (ZOMBIE VARIANT)</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/comics/22300</string>
						</dict>
						<dict>
							<key>name</key>
							<string>Avengers: The Initiative (2007) #19</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/comics/22506</string>
						</dict>
						<dict>
							<key>name</key>
							<string>Deadpool (1997) #44</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/comics/8500</string>
						</dict>
						<dict>
							<key>name</key>
							<string>Marvel Premiere (1972) #35</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/comics/10223</string>
						</dict>
						<dict>
							<key>name</key>
							<string>Marvel Premiere (1972) #36</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/comics/10224</string>
						</dict>
						<dict>
							<key>name</key>
							<string>Marvel Premiere (1972) #37</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/comics/10225</string>
						</dict>
					</array>
					<key>returned</key>
					<integer>12</integer>
				</dict>
				<key>description</key>
				<string></string>
				<key>events</key>
				<dict>
					<key>available</key>
					<integer>1</integer>
					<key>collectionURI</key>
					<dict>
						<key>relative</key>
						<string>http://gateway.marvel.com/v1/public/characters/1011334/events</string>
					</dict>
					<key>items</key>
					<array>
						<dict>
							<key>name</key>
							<string>Secret Invasion</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/events/269</string>
						</dict>
					</array>
					<key>returned</key>
					<integer>1</integer>
				</dict>
				<key>id</key>
				<integer>1011334</integer>
				<key>modified</key>
				<date>2014-04-29T18:18:17Z</date>
				<key>name</key>
				<string>3-D Man</string>
				<key>resourceURI</key>
				<dict>
					<key>relative</key>
					<string>http://gateway.marvel.com/v1/public/characters/1011334</string>
				</dict>
				<key>series</key>
				<dict>
					<key>available</key>
					<integer>3</integer>
					<key>collectionURI</key>
					<dict>
						<key>relative</key>
						<string>http://gateway.marvel.com/v1/public/characters/1011334/series</string>
					</dict>
					<key>items</key>
					<array>
						<dict>
							<key>name</key>
							<string>Avengers: The Initiative (2007 - 2010)</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/series/1945</string>
						</dict>
						<dict>
							<key>name</key>
							<string>Deadpool (1997 - 2002)</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/series/2005</string>
						</dict>
						<dict>
							<key>name</key>
							<string>Marvel Premiere (1972 - 1981)</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/series/2045</string>
						</dict>
					</array>
					<key>returned</key>
					<integer>3</integer>
				</dict>
				<key>stories</key>
				<dict>
					<key>available</key>
					<integer>21</integer>
					<key>collectionURI</key>
					<dict>
						<key>relative</key>
						<string>http://gateway.marvel.com/v1/public/characters/1011334/stories</string>
					</dict>
					<key>items</key>
					<array>
						<dict>
							<key>name</key>
							<string>Cover #19947</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/stories/19947</string>
						</dict>
						<dict>
							<key>name</key>
							<string>The 3-D Man!</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/stories/19948</string>
						</dict>
						<dict>
							<key>name</key>
							<string>Cover #19949</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/stories/19949</string>
						</dict>
						<dict>
							<key>name</key>
							<string>The Devil's Music!</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/stories/19950</string>
						</dict>
						<dict>
							<key>name</key>
							<string>Cover #19951</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/stories/19951</string>
						</dict>
						<dict>
							<key>name</key>
							<string>Code-Name:  The Cold Warrior!</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/stories/19952</string>
						</dict>
						<dict>
							<key>name</key>
							<string>AVENGERS: THE INITIATIVE (2007) #14</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/stories/47184</string>
						</dict>
						<dict>
							<key>name</key>
							<string>Avengers: The Initiative (2007) #14 - Int</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/stories/47185</string>
						</dict>
						<dict>
							<key>name</key>
							<string>AVENGERS: THE INITIATIVE (2007) #15</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/stories/47498</string>
						</dict>
						<dict>
							<key>name</key>
							<string>Avengers: The Initiative (2007) #15 - Int</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/stories/47499</string>
						</dict>
						<dict>
							<key>name</key>
							<string>AVENGERS: THE INITIATIVE (2007) #16</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/stories/47792</string>
						</dict>
						<dict>
							<key>name</key>
							<string>Avengers: The Initiative (2007) #16 - Int</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/stories/47793</string>
						</dict>
						<dict>
							<key>name</key>
							<string>AVENGERS: THE INITIATIVE (2007) #17</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/stories/48361</string>
						</dict>
						<dict>
							<key>name</key>
							<string>Avengers: The Initiative (2007) #17 - Int</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/stories/48362</string>
						</dict>
						<dict>
							<key>name</key>
							<string>AVENGERS: THE INITIATIVE (2007) #18</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/stories/49103</string>
						</dict>
						<dict>
							<key>name</key>
							<string>Avengers: The Initiative (2007) #18 - Int</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/stories/49104</string>
						</dict>
						<dict>
							<key>name</key>
							<string>Avengers: The Initiative (2007) #18, Zombie Variant - Int</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/stories/49106</string>
						</dict>
						<dict>
							<key>name</key>
							<string>AVENGERS: THE INITIATIVE (2007) #19</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/stories/49888</string>
						</dict>
						<dict>
							<key>name</key>
							<string>Avengers: The Initiative (2007) #19 - Int</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/stories/49889</string>
						</dict>
						<dict>
							<key>name</key>
							<string>Avengers: The Initiative (2007) #14, Spotlight Variant - Int</string>
							<key>resourceURI</key>
							<string>http://gateway.marvel.com/v1/public/stories/54371</string>
						</dict>
					</array>
					<key>returned</key>
					<integer>20</integer>
				</dict>
				<key>thumbnail</key>
				<dict>
					<key>extension</key>
					<string>jpg</string>
					<key>path</key>
					<string>http://i.annihil.us/u/prod/marvel/i/mg/c/e0/535fecbbb9784</string>
				</dict>
				<key>urls</key>
				<array>
					<dict>
						<key>type</key>
						<string>detail</string>
						<key>url</key>
						<dict>
							<key>relative</key>
							<string>http://marvel.com/comics/characters/1011334/3-d_man?utm_campaign=apiRef&amp;utm_source=746e3520c1b5a62cb96c269427d784a3</string>
						</dict>
					</dict>
					<dict>
						<key>type</key>
						<string>wiki</string>
						<key>url</key>
						<dict>
							<key>relative</key>
							<string>http://marvel.com/universe/3-D_Man_(Chandler)?utm_campaign=apiRef&amp;utm_source=746e3520c1b5a62cb96c269427d784a3</string>
						</dict>
					</dict>
					<dict>
						<key>type</key>
						<string>comiclink</string>
						<key>url</key>
						<dict>
							<key>relative</key>
							<string>http://marvel.com/comics/characters/1011334/3-d_man?utm_campaign=apiRef&amp;utm_source=746e3520c1b5a62cb96c269427d784a3</string>
						</dict>
					</dict>
				</array>
			</dict>
		</array>
		<key>total</key>
		<integer>1</integer>
	</dict>
	<key>etag</key>
	<string>f0f50f72d6ce5fc336cf70a7c2be616ce78215c8</string>
	<key>status</key>
	<string>Ok</string>
</dict>
</plist>
