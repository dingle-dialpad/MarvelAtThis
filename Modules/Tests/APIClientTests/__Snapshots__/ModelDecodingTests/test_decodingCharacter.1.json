{
  "data" : {
    "count" : 1,
    "cursor" : {
      "limit" : 20,
      "offset" : 0
    },
    "results" : [
      {
        "comics" : {
          "available" : 12,
          "collectionURI" : "http:\/\/gateway.marvel.com\/v1\/public\/characters\/1011334\/comics",
          "items" : [
            {
              "name" : "Avengers: The Initiative (2007) #14",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/comics\/21366"
            },
            {
              "name" : "Avengers: The Initiative (2007) #14 (SPOTLIGHT VARIANT)",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/comics\/24571"
            },
            {
              "name" : "Avengers: The Initiative (2007) #15",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/comics\/21546"
            },
            {
              "name" : "Avengers: The Initiative (2007) #16",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/comics\/21741"
            },
            {
              "name" : "Avengers: The Initiative (2007) #17",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/comics\/21975"
            },
            {
              "name" : "Avengers: The Initiative (2007) #18",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/comics\/22299"
            },
            {
              "name" : "Avengers: The Initiative (2007) #18 (ZOMBIE VARIANT)",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/comics\/22300"
            },
            {
              "name" : "Avengers: The Initiative (2007) #19",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/comics\/22506"
            },
            {
              "name" : "Deadpool (1997) #44",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/comics\/8500"
            },
            {
              "name" : "Marvel Premiere (1972) #35",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/comics\/10223"
            },
            {
              "name" : "Marvel Premiere (1972) #36",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/comics\/10224"
            },
            {
              "name" : "Marvel Premiere (1972) #37",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/comics\/10225"
            }
          ],
          "returned" : 12
        },
        "description" : "",
        "events" : {
          "available" : 1,
          "collectionURI" : "http:\/\/gateway.marvel.com\/v1\/public\/characters\/1011334\/events",
          "items" : [
            {
              "name" : "Secret Invasion",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/events\/269"
            }
          ],
          "returned" : 1
        },
        "id" : 1011334,
        "modified" : "2014-04-29T18:18:17Z",
        "name" : "3-D Man",
        "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/characters\/1011334",
        "series" : {
          "available" : 3,
          "collectionURI" : "http:\/\/gateway.marvel.com\/v1\/public\/characters\/1011334\/series",
          "items" : [
            {
              "name" : "Avengers: The Initiative (2007 - 2010)",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/series\/1945"
            },
            {
              "name" : "Deadpool (1997 - 2002)",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/series\/2005"
            },
            {
              "name" : "Marvel Premiere (1972 - 1981)",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/series\/2045"
            }
          ],
          "returned" : 3
        },
        "stories" : {
          "available" : 21,
          "collectionURI" : "http:\/\/gateway.marvel.com\/v1\/public\/characters\/1011334\/stories",
          "items" : [
            {
              "name" : "Cover #19947",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/stories\/19947"
            },
            {
              "name" : "The 3-D Man!",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/stories\/19948"
            },
            {
              "name" : "Cover #19949",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/stories\/19949"
            },
            {
              "name" : "The Devil's Music!",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/stories\/19950"
            },
            {
              "name" : "Cover #19951",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/stories\/19951"
            },
            {
              "name" : "Code-Name:  The Cold Warrior!",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/stories\/19952"
            },
            {
              "name" : "AVENGERS: THE INITIATIVE (2007) #14",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/stories\/47184"
            },
            {
              "name" : "Avengers: The Initiative (2007) #14 - Int",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/stories\/47185"
            },
            {
              "name" : "AVENGERS: THE INITIATIVE (2007) #15",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/stories\/47498"
            },
            {
              "name" : "Avengers: The Initiative (2007) #15 - Int",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/stories\/47499"
            },
            {
              "name" : "AVENGERS: THE INITIATIVE (2007) #16",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/stories\/47792"
            },
            {
              "name" : "Avengers: The Initiative (2007) #16 - Int",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/stories\/47793"
            },
            {
              "name" : "AVENGERS: THE INITIATIVE (2007) #17",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/stories\/48361"
            },
            {
              "name" : "Avengers: The Initiative (2007) #17 - Int",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/stories\/48362"
            },
            {
              "name" : "AVENGERS: THE INITIATIVE (2007) #18",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/stories\/49103"
            },
            {
              "name" : "Avengers: The Initiative (2007) #18 - Int",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/stories\/49104"
            },
            {
              "name" : "Avengers: The Initiative (2007) #18, Zombie Variant - Int",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/stories\/49106"
            },
            {
              "name" : "AVENGERS: THE INITIATIVE (2007) #19",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/stories\/49888"
            },
            {
              "name" : "Avengers: The Initiative (2007) #19 - Int",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/stories\/49889"
            },
            {
              "name" : "Avengers: The Initiative (2007) #14, Spotlight Variant - Int",
              "resourceURI" : "http:\/\/gateway.marvel.com\/v1\/public\/stories\/54371"
            }
          ],
          "returned" : 20
        },
        "thumbnail" : {
          "extension" : "jpg",
          "path" : "http:\/\/i.annihil.us\/u\/prod\/marvel\/i\/mg\/c\/e0\/535fecbbb9784"
        },
        "urls" : [
          {
            "type" : "detail",
            "url" : "http:\/\/marvel.com\/comics\/characters\/1011334\/3-d_man?utm_campaign=apiRef&utm_source=746e3520c1b5a62cb96c269427d784a3"
          },
          {
            "type" : "wiki",
            "url" : "http:\/\/marvel.com\/universe\/3-D_Man_(Chandler)?utm_campaign=apiRef&utm_source=746e3520c1b5a62cb96c269427d784a3"
          },
          {
            "type" : "comiclink",
            "url" : "http:\/\/marvel.com\/comics\/characters\/1011334\/3-d_man?utm_campaign=apiRef&utm_source=746e3520c1b5a62cb96c269427d784a3"
          }
        ]
      }
    ],
    "total" : 1
  },
  "metadata" : {
    "code" : 200,
    "etag" : "f0f50f72d6ce5fc336cf70a7c2be616ce78215c8",
    "status" : "Ok"
  }
}